qa/rpc-tests/wallet-hd.py

```diff
@@ -45,6 +49,9 @@ def run_test (self):
         num_hd_adds = 300
         for _ in range(num_hd_adds):
             hd_add = self.nodes[1].getnewaddress()
+            hd_info = self.nodes[1].validateaddress(hd_add)
+            assert_equal(hd_info["hdkeypath"], "m/0'/0'/"+str(_+1)+"'")
```

Comment:
Nit: underscore is usually used for unused loop variables. Mind to change it to a single char?